src/main/java/it/gov/pagopa/gpd/payments/pull/models/PaymentNotice.java

```diff
@@ -20,7 +19,12 @@
 @ToString
 public class PaymentNotice implements Serializable {
 
-    @Schema(description = "Unique ID of the Debt Position (for positions originating from ACA the IUPD starts with ACA_)",
+    /**
+	 * 
+	 */
+	private static final long serialVersionUID = -5951473285697199137L;
+	
+	@Schema(description = "Unique ID of the Debt Position (for positions originating from ACA)",
```

Comment:
> for positions originating from ACA

what does it mean? 
for positions originating from `ACA` the IUPD starts with `ACA_` will no longer be valid after the db is updated.
Does it still make sense to emphasize the fact that IUPD will be different? Maybe yes, but the description must not be updated.

Reply:
I suggest to simplify the description with: `Unique ID of the Debt Position`

Reply:
For me it no longer made sense to emphasize that aca positions are identified by putting ACA_ as the IUPD prefix (since now there is a specific field).
But it was a personal consideration, i can safely restore the original description :)

Reply:
so this part `(for positions originating from ACA)` should be removed